addons/project/views/res_config_settings_views.xml

```diff
@@ -9,9 +9,6 @@
                 <xpath expr="//form" position="inside">
                     <app data-string="Project" string="Project" name="project" groups="project.group_project_manager">
                         <block title="Tasks Management" id="tasks_management">
-                            <setting help="Split your tasks to organize your work into sub-milestones">
-                                <field name="group_subtask_project"/>
-                            </setting>
```

Comment:
It is better to make it invisible instead of remove it in stable. 😇 
```suggestion
                            <setting invisible="1" help="Split your tasks to organize your work into sub-milestones">
                                <field name="group_subtask_project"/>
                            </setting>
```

It is just to be sure to not break an eventual xpath in a customer db. 🙂

Reply:
Done